edx_solutions_api_integration/users/views.py

```diff
@@ -1493,8 +1493,18 @@ def get_queryset(self):
         user = get_user_from_request_params(self.request, self.kwargs)
         if not user:
             return []
-
-        return user.organizations.all()
+        main_user_organization = None
```

Comment:
Lines 1469-1502 seems duplicate code(used in serializer as well).
can we remove this duplication as both are related with each other.